src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -1,65 +1,60 @@
-import React, { useCallback, useMemo, useState } from "react";
-import { Button, Nav } from "react-bootstrap";
-import { Link } from "react-router-dom";
+import React, { useMemo, useState } from "react";
+import { Nav } from "react-bootstrap";
 import classNames from "classnames";
 
-import AdvancedSettings from "pages/Admin/AdvancedSettings";
-import BasicInfo from "pages/Admin/BasicInfo";
-import EntranceExperience from "pages/Admin/EntranceExperience";
-import VenueDetails from "pages/Admin/Venue/Details";
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
+import { useUser } from "hooks/useUser";
+import { useAdminVenues } from "hooks/useAdminVenues";
 
-import { Venue_v2 } from "types/venues";
+import { LoadingPage } from "components/molecules/LoadingPage";
 
 import "./AdminVenueView.scss";
 
 export enum AdminVenueTab {
-  dashboard = "dashboard",
-  basicInfo = "basic_info",
-  entranceExperience = "entrance_experience",
-  advancedMapSettings = "advanced_map_settings",
+  spaces = "spaces",
+  timing = "timing",
+  run = "run",
 }
 
 const adminVenueTabs: Record<AdminVenueTab, String> = {
-  [AdminVenueTab.basicInfo]: "Start",
-  [AdminVenueTab.entranceExperience]: "Entrance",
-  [AdminVenueTab.advancedMapSettings]: "Advanced",
-  [AdminVenueTab.dashboard]: "Dashboard",
+  [AdminVenueTab.spaces]: "Spaces",
+  [AdminVenueTab.timing]: "Timing",
+  [AdminVenueTab.run]: "Run",
 };
 
-const DEFAULT_TAB = AdminVenueTab.dashboard;
+const DEFAULT_TAB = AdminVenueTab.spaces;
 
-export interface AdminVenueViewProps {
-  venue: Venue_v2;
-}
+export const AdminVenueView: React.FC = () => {
+  const { user } = useUser();
+  useAdminVenues(user?.uid);
 
-export const AdminVenueView: React.FC<AdminVenueViewProps> = ({ venue }) => {
+  const venueId = useVenueId();
   const [selectedTab, setSelectedTab] = useState<string>(DEFAULT_TAB);
 
-  const selectDefaultTab = useCallback(() => {
-    setSelectedTab(DEFAULT_TAB);
-  }, []);
+  const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
```

Comment:
‘Next generation’. The idea was basically for this function to replace the legacy useCurrentVenue stuff as it uses suboptimal patterns, but without needing to refactor it across the entire codebase.

Eventually we want to be able to remove the legacy version, and then this will be the only version (at which point we can rename it)

Reply:
Oh well, completely opposite from what I suspected.

"The letters NG, shown enclosed in a square, which are an acronym for the words No Good." - https://emojipedia.org/ng-button/